src/assets/i18n/pt-BR.json5

```diff
@@ -27,14 +27,14 @@
   "500.help": "O servidor está temporariamente impossibilitado de atender sua solicitação devido a um período de manutenção ou problemas no servidor. Por favor, tente novamente mais tarde.",
 
   // "500.link.home-page": "Take me to the home page",
-  "500.link.home-page": "Leve-me para a página inicial",
+  "500.link.home-page": "Retornar à página inicial",
```

Comment:
The same as in previous comments... It makes more sense to have:
`Acessar a página inicial`